### ISSUE TITLE
CTA after walkthrough may be confusing

### ISSUE THREAD

**Issue by kntsoriano** [exercism/interactive-cli-walkthrough#77]
After finishing the walkthrough, you are shown the following:

<img width="1280" alt="screen shot 2018-11-30 at 4 22 02 pm" src="https://user-images.githubusercontent.com/1901520/49297998-8f4cbc80-f4bc-11e8-8e77-736b2dd6e3c2.png">

In the context of the walkthrough being on a modal in the exercise page, I think it might be confusing that we are asking the user to choose and exercise from their dashboard. This should instruct them to close the modal, instead.

If we open the walkthrough in a new tab, we see the following after finishing the walkthrough:

<img width="1278" alt="screen shot 2018-11-30 at 4 22 42 pm" src="https://user-images.githubusercontent.com/1901520/49298095-ccb14a00-f4bc-11e8-8be5-dd9ca3eeb213.png">

This sounds correct because the page is not within the context of an exercise, but come to think of it, in order to have gotten to this page, you should still have been in a context of an exercise.

Would it make sense to change the installation complete message in order to say that they can now continue on with the exercise they've chosen?

**Comment by kytrinyx** [comment]
> in order to have gotten to this page, you should still have been in a context of an exercise.

You can get to the walkthrough via the footer:

<img width="308" alt="screen shot 2018-12-01 at 08 39 50" src="https://user-images.githubusercontent.com/276834/49330511-af27c300-f544-11e8-8a38-51081556db70.png">

I agree that if you're within the context of an exercise you should close the modal, but that the dashboard link is useful if not. I'm not sure how to do that, though!

**Comment by kntsoriano** [comment]
I didn't realize that we could access the walkthrough from the footer. How could we fix this? Or is it even a problem we want to fix?

I can think of the following ways:
1. Write copy that accommodates for both cases by making it more general. I'm not particularly fond of this solution as this doesn't help our user in any way.
2. Restructure the website links in order to for the walkthrough to link to only one CTA at the end.
3. Write two different versions of the last page of the walkthrough. I'm unsure this fixes the underlying problem, which is that the walkthrough is used in two different places than it is intended for. If we wanted to keep the walkthrough separate, I think we need to understand what its capabilities and limits are.

**Comment by kytrinyx** [comment]
@kntsoriano Agreed. Let's discuss this with @iHiD and @nicolechalmers when the holidays are over. I think we need to take a step back and figure out exactly what problems we'd like to solve here.
